plugins/fuel.py

```diff
@@ -1,102 +1,74 @@
-#!/usr/bin/env python2
-# -*- coding: utf-8 -*-
+from irc3.plugins.command import command
 
-import urllib2
-import json
-from time import time
-
-from plugins.plugin import Plugin
-
-from bytebot_config import BYTEBOT_HTTP_TIMEOUT, BYTEBOT_HTTP_MAXSIZE
 from bytebot_config import BYTEBOT_PLUGIN_CONFIG
+from irc3 import asyncio
+import json
+import aiohttp
 
+@command(permission="view")
+@asyncio.coroutine
+def fuel(bot, mask, target, args):
+    """Show the current fuel price for fuelsort
 
-class fuel(Plugin):
+        %%fuel [<fuelsort>]...
```

Comment:
Maybe call that fueltype? `fuelsort` kinda sounds like a hot new sorting algorithm